packages/integrations/image/src/config.ts

```diff
@@ -0,0 +1,3 @@
+export const PKG_NAME = '@astrojs/image';
+export const ROUTE_PATTERN = '/_image';
+export const OUTPUT_DIR = '/_image';
```

Comment:
Such a small nit but I tend to call this file `constants.ts`. `config` is a bit overloaded for our use cases.

Reply:
Yep great catch, I had named it `config` planning on using it to also normalize integration config and never circled back to it